### PR TITLE
[WIP] Update name and ssh key

### PR DESCRIPTION
**Pull request by binaryberry**
I have a new laptop therefore a new ssh key, and I got married so my name has
changed. I'm taking this opportunity to update my name and email. Happy to keep
my maiden name and email if there's a chance the new data will cause issues.

### CODE REVIEW

File: modules/users/manifests/tatianasoukiassian.pp [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Creates the tatianasoukiassian user
 class users::tatianasoukiassian {
```
Comment (surminus) [inline modules/users/manifests/tatianasoukiassian.pp]: This also needs to be updated.

Comment (binaryberry) [inline modules/users/manifests/tatianasoukiassian.pp]: ok! My LDAP is still tatianasoukiassian. Might that cause an issue?

**Comment by cbaines** [comment]
I've added WIP to the title as I believe this PR will be dormant for a little while, and this will get the Seal off my back.

**Comment by deanwilson** [comment]
Can we bring this one back to life or close it off as it's been around for a while. The tests fail as the name isn't consistently changed everywhere, and if we do change it we might have a little discovery as to what breaks. Which might be worth doing incase someone else gets married. Hopefully it's just changing the local machines ssh config to use the new name.

**Comment by surminus** [comment]
I'm going to close this PR, please feel free to raise a new one if you wish to update your name.

**Pull request closed without merging**